CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v6.5.0
+
+_May 19, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Introduce filtering on column headers for `DataGridPro`:
```

Comment:
```suggestion
- 💫 Introduce filtering on column headers for `DataGridPro` and `DataGridPremium`:
```

Or is it totally obvious ?

Reply:
`- 💫 Introduce filtering on column headers for commercial users` 

could also be an option 🤷‍♂️